exonum/src/api/node/public/explorer.rs

```diff
@@ -400,6 +470,18 @@ impl ExplorerApi {
                 let blockchain = self.blockchain.clone();
                 move |query| Self::block(blockchain.snapshot().as_ref(), query)
             })
+            .endpoint("v1/call_status/transaction", {
+                let blockchain = self.blockchain.clone();
+                move |query| Self::transaction_status(blockchain.snapshot().as_ref(), query)
```

Comment:
Nit: `&blockchain.snapshot()` instead of `.as_ref()` would be more concise. Again, could be solved separately.